src/cpr_sdk/version.py

```diff
@@ -1,5 +1,5 @@
 _MAJOR = "1"
-_MINOR = "1"
+_MINOR = "3"
```

Comment:
Why was 2 skipped?

Reply:
Only because I used it on this one: https://github.com/climatepolicyradar/cpr-sdk/pull/76/files

Reply:
haha, of course, sorry!